internal/p2p/pex/reactor.go

```diff
@@ -221,7 +221,7 @@ func (r *ReactorV2) handlePexMessage(envelope p2p.Envelope) error {
 		}
 
 		// check the size of the response
-		if len(msg.Addresses) > int(maxAddresses) {
+		if len(msg.Addresses) > int(maxGetSelection) {
```

Comment:
This looks like a change in semantics, not just value. Why the change?

Reply:
This change is specific to mamaki, where we have an existing large network of mixed legacy and new p2p stacks. We need it so that nodes using the new p2p stack can exchange peers with nodes that have over 100 peers and are using the legacy p2p stack (most nodes on mamaki)

the change linked in the issue fixes this in tendermint, but it doesn't fix this for existing networks. 

As far as semantically, this is just the variable that is used by the legacy pex, so we use that here.